run_tests.sh

```diff
@@ -2,5 +2,6 @@
 set -e
 VIRTUALENV=/opt/env/atmo
 SCRIPT_DIR=$(dirname `readlink -f $0`)
-source $VIRTUALENV/bin/activate
+#source $VIRTUALENV/bin/activate
+source $SCRIPT_DIR/.tox/py27dj19/bin/activate
```

Comment:
@knikolla So I don't have to do any `source` command here right? It was sourced with `atmo` but not the tox virtualenv. I am afraid of doing a `source` here would switch the env back to `atmo`.